driver/kubernetes/manifest/manifest.go

```diff
@@ -148,21 +148,21 @@ func NewDeployment(opt *DeploymentOpt) (d *appsv1.Deployment, c []*corev1.Config
 			Data: cfg.files,
 		}
 
-		d.Spec.Template.Spec.Containers[0].VolumeMounts = []corev1.VolumeMount{{
-			Name:      cfg.name,
+		d.Spec.Template.Spec.Containers[0].VolumeMounts = append(d.Spec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
+			Name:      cc.Name,
```

Comment:
I think it would be always empty as this field is not init. Should keep `cfg.Name`

Reply:
It is initialized here: https://github.com/docker/buildx/pull/2462/files#diff-ad9f131660b45257e1b16c6d5be47cfc1e4bb74f0322fbc6619afedba317288cR145